x-pack/plugins/observability_solution/observability_ai_assistant/server/service/client/adapters/openai_adapter.ts

```diff
@@ -44,7 +44,16 @@ export const createOpenAiAdapter: LlmApiAdapterFactory = ({
           })
       );
 
-      const functionsForOpenAI = functions;
+      const functionsForOpenAI = functions?.map((fn) => ({
+        parameters: merge(
+          {
+            type: 'object',
+            properties: {},
+          },
+          fn.parameters
+        ),
+        ...fn,
```

Comment:
yes, good catch. it doesn't matter in practice because _if_ `fn.parameters` is defined it has at least the same shape as `{ type: 'object' }`, but it's silly code, and I'll fix it. Thanks